Assets/Vimeo/Scripts/Services/VimeoApi.cs

```diff
@@ -51,6 +57,12 @@ public enum CommentMode
 
         public void Start()
         {
+            if (uuidGenerator == null) {
```

Comment:
Seems like this should exist inside of `PlayLogging`. Can we create an instance of that class that auto generates the session? that way you dont have to worry about passing all these params in.

Reply:
Great idea, ty! - https://github.com/vimeo/vimeo-unity-sdk/pull/83/commits/00177a4f5573de17102e51cff15d1cf801b1f9db